ui/components/app/multilayer-fee-message/multi-layer-fee-message.test.js

```diff
@@ -26,6 +26,30 @@ describe('Multi layer fee message', () => {
     expect(container).toMatchSnapshot();
   });
 
+  it('should match snapshot when balance and token price checker is disabled', () => {
+    const storeWithPriceCheckerDisabled = configureStore({
+      ...mockState,
+      metamask: {
+        ...mockState.metamask,
+        useCurrencyRateCheck: false,
+      },
+    });
+
+    const { container } = renderWithProvider(
+      <MultilayerFeeMessage
+        transaction={{
+          txParams: {
+            value: '0x38d7ea4c68000',
+          },
+        }}
+        layer2fee="0x4e3b29200"
+        nativeCurrency="ETH"
+      />,
+      storeWithPriceCheckerDisabled,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
   it('should containe fee values', () => {
```

Comment:
[out-of-scope but since it's here]
```suggestion
  it('should contain fee values', () => {
```